python/ray/tune/suggest/ax.py

```diff
@@ -190,20 +185,26 @@ def _setup_experiment(self):
                 objective_name=self._metric,
                 parameter_constraints=self._parameter_constraints,
                 outcome_constraints=self._outcome_constraints,
-                minimize=self._mode != "max")
+                minimize=self._mode != "max"
```

Comment:
I'm wondering if in this case it should be required that the mode is not none? Otherwise we are defaulting to a minimization problem –– not sure whether that's okay

Reply:
Good catch, we should introduce a check above this block (after the check for `self._space`)